### ISSUE TITLE
Support for queueing up of multiple requests

### ISSUE THREAD

**Issue by tarelli** [VirtualFlyBrain/geppetto-vfb#183]
- [x] Debug to find what breaks
- [x] Fix what breaks (might require some new functionality to implement but not clear yet)
- [x] Add VFB specific casper test to send multiple requests

**Comment by tarelli** [comment]
@Robbie1977 what's the current status of casper's if we want to add one?

**Comment by Robbie1977** [comment]
They run but all fail for various reasons: https://travis-ci.org/VirtualFlyBrain/geppetto-vfb/jobs/390979325

**Comment by ddelpiano** [comment]
fixes applied to geppetto-vfb extension's repo, branch fix/queueing_multiple_requests.
Still checking few things failing in the capserjs test, once done moving to review.

@Robbie1977 I will apply also a change in the vfb.js script in the VFB2 repo, I would like to replace the addVfbIds call with addVfbId if you are ok with that.
Tested locally on my side is working fine now and I reworked the buffer logic in the component to use both the calls, if you want to keep that call it's fine but be aware will be called only at startup, everything else requested through the UI is now using addVfbId.
